cmd/cronosd/cmd/root.go

```diff
@@ -203,10 +205,48 @@ func txCommand() *cobra.Command {
 	return cmd
 }
 
+// AppConfig helps to override default appConfig template and configs.
+// return "", nil if no custom configuration is required for the application.
+func AppConfig(denom string) (string, interface{}) {
+	// Optionally allow the chain developer to overwrite the SDK's default
+	// server config.
+	srvCfg := sdkserverconfig.DefaultConfig()
+
+	// The SDK's default minimum gas price is set to "" (empty value) inside
+	// app.toml. If left empty by validators, the node will halt on startup.
+	// However, the chain developer can set a default app.toml value for their
+	// validators here.
+	//
+	// In summary:
+	// - if you leave srvCfg.MinGasPrices = "", all validators MUST tweak their
+	//   own app.toml config,
+	// - if you set srvCfg.MinGasPrices non-empty, validators CAN tweak their
+	//   own app.toml to override, or use this default value.
+	//
+	// In ethermint, we set the min gas prices to 0.
+	if denom != "" {
+		srvCfg.MinGasPrices = "0" + denom
+	}
+
+	srvCfg.IAVLDisableFastNode = false
```

Comment:
I think the `iavl-disable-fastnode` in `app.toml` is default to `false`.

Reply:
you're right I'll remove this unnecessary change

Reply:
I'll close this PR as the result.